### DISCUSSION TITLE
Control over installed wallets after EIP 6963 update

### DISCUSSION THREAD

**Discussion by umershaikh123** [rainbow-me/rainbowkit#2262] [Q&A] [answered]
### Rainbow kit doc EIP 6963 changes 
[RainbowKit Changes](https://www.rainbowkit.com/docs/migration-guide#rainbowkit-changes)
1. Improved behavior for EIP-6963 wallets

Wallets that support the new EIP-6963 connection standard (including Rainbow, MetaMask, and more) will now automatically appear in an Installed section during the Connect Wallet experience. This ensures that users can always find their favorite wallets and connect to dApps without conflicts or fallback buttons.
 
### issue
i want to have control over which wallets show up in the installed section but EIP 6963 shows all wallets installed in the connect wallet popup , is there a way to remove some wallet from the installed section with EIP 6963 enabled ?

for example , i want to don't certain wallets to show up in the installed section . I already read the custom wallet list and wagmi create config but found nothing 
![image](https://github.com/user-attachments/assets/d395a487-786d-44e0-aaa6-71428099cfbd)

**Comment by DanielSinclair** [comment, marked as answer]
You can disable `multiInjectedProviderDiscovery` in a custom `createConfig` for Wagmi if you really need to, but it is generally discouraged as surfacing connected wallets will be more intuitive for users. https://wagmi.sh/react/api/createConfig#multiinjectedproviderdiscovery
